### PR TITLE
Code cleanup for Microsoft.Data.Sqlite

### PR DESCRIPTION
**Pull request by ajcvickers**

### CODE REVIEW

File: src/Microsoft.Data.Sqlite.Core/SqliteConnection.cs [hunk 1]
```diff
@@ -821,8 +825,9 @@ public virtual void BackupDatabase(SqliteConnection destination, string destinat
                 SqliteException.ThrowExceptionForRC(rc, _db);
             }
 
-            _aggregates ??= new Dictionary<(string, int), (int, object, delegate_function_aggregate_step, delegate_function_aggregate_final)>(
-                FunctionsKeyComparer.Instance);
+            _aggregates ??=
```
Comment (bricelam) [inline src/Microsoft.Data.Sqlite.Core/SqliteConnection.cs:828]: 😬

Comment (ajcvickers) [reply src/Microsoft.Data.Sqlite.Core/SqliteConnection.cs:828]: Where would you have expected this to split the line?

Comment (bricelam) [reply src/Microsoft.Data.Sqlite.Core/SqliteConnection.cs:828]: It’s the dangling `=` issue we discussed previously.

Comment (ajcvickers) [reply src/Microsoft.Data.Sqlite.Core/SqliteConnection.cs:828]: Ah, right. I'll check if this is configured differently for `??=`.

File: src/Microsoft.Data.Sqlite.Core/Extensions/TypeExtensions.cs [hunk 2]
```diff
@@ -1,7 +1,7 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
-using System.Reflection;
+
```
Comment (Saibamen) [inline src/Microsoft.Data.Sqlite.Core/Extensions/TypeExtensions.cs:4]: Line should be delete?

**Pull request merged**